invenio_app_rdm/communities_ui/views/ui.py

```diff
@@ -94,6 +94,16 @@ def create_ui_blueprint(app):
         view_func=communities_home,
     )
 
+    blueprint.add_url_rule(
+        routes["community-static-page"],
+        view_func=community_static_page,
+    )
+
+    blueprint.add_url_rule(
```

Comment:
shelf: this would be simplified in the future, to use the same registration flow as `invenio-pages` does in order not to manually register the url on the blueprint.